tests-ng/test_users.py

```diff
@@ -23,3 +25,25 @@ def test_no_extra_home_directories(expected_users):
     entries = os.listdir("/home")
     unexpected = [e for e in entries if e not in expected_users]
     assert not unexpected, f"Unexpected entries in /home: {entries}"
+
+@pytest.mark.skipif((shutil.which("sudo")) is None, reason="sudo does not exist")
+@pytest.mark.booted
+@pytest.mark.root
+def test_users_sudo_capability(expected_users, shell: ShellRunner):
+    for entry in pwd.getpwall():
+        output = shell(f"sudo --list --other-user={entry.pw_name}", capture_output=True)
```

Comment:
https://github.com/gardenlinux/gardenlinux/pull/3634#discussion_r2418852340 still needs to be addressed (moving the parsing to a fixture or helper function)